docs/modules/ROOT/pages/guide-metatx.adoc

```diff
@@ -0,0 +1,349 @@
+
+  fs.writeFileSync('deploy.json', JSON.stringify({
+    MinimalForwarder: forwarder.address,
+    Registry: registry.address,
+  }, null, 2));
+
+  console.log(`MinimalForwarder: ${forwarder.address}\nRegistry: ${registry.address}`);
+}
+
+if (require.main === module) {
+  main().then(() => process.exit(0))
+    .catch(error => { console.error(error); process.exit(1); });
+}
+
+```
+
+This code pulls the Relayer's credentials from the local file along with the artifacts for the `Registry` and `MinimalForwarder` contracts and uses ethers.js to deploy. The relevant addresses of these contracts are saved to the local file `deploy.json`.
+
+In the demo app, run this script with `yarn deploy`
+
+Note: As the owners of the Relayer, are the paying for the gas, so it is necessary to keep the Relayer funded sufficiently to cover future transactions. For Defender users on testnets, Relayer balances are auto-topped as necessary to ensure transactions go through.
+
+
+[[create-autotask]]
+== Create Autotask via API
+
+Although you can create an Autotask using the Defender web client, it may be more convenient to use a simple script that makes use of https://www.npmjs.com/package/defender-autotask-client[Defender's API].
+
+First you'll need to install the package: `npm install defender-autotask-client`
+
+Create a script like the following to create an Autotask and upload its code:
+
+```
+const { AutotaskClient } = require('defender-autotask-client');
+const {readFileSync, appendFileSync} = require('fs');
+
+async function main() {
+  require('dotenv').config();
+  const {relayer: {relayerId}} = JSON.parse(readFileSync('./relay.json'))
+  const { TEAM_API_KEY: apiKey, TEAM_API_SECRET: apiSecret } = process.env;
+  const client = new AutotaskClient({ apiKey, apiSecret });
+  const {autotaskId } = await client.create({
+    name: "Relay MetaTX",
+    encodedZippedCode: await client.getEncodedZippedCodeFromFolder('./build/relay'),
+    relayerId: relayerId, 
+    trigger: {
+      type: 'webhook'
+    },
+    paused: false
+  });
+  console.log("Autotask created with ID ", autotaskId);
+  appendFileSync('.env', `\nAUTOTASK_ID="${autotaskId}"`, function (err) {
+    if (err) throw err;
+ });
+}
+
+if (require.main === module) {
+  main().then(() => process.exit(0))
+    .catch(error => { console.error(error); process.exit(1); });
+}
+```
+
+Next, create the Autotask itself. The main idea here is to supply the necessary logic for telling the Relayer to send a transaction to the forwarder contract, supplying the signer's address. The Autotask will get triggered by a call to its webhook from the dapp.
+
+Due to the tight relationship between components, the Relayer credentials are securely available to the Autotask simply by instantiating a new provider and signer. Note that the address of the `MinimalForwarder` and the `Registry` contract must be supplied.
```

Comment:
Good point - why are we storing the relayer secret key to the local env file then?